Enigmatry.Entry.Blueprint.Api.Tests/Infrastructure/ApiWebApplicationFactory.cs

```diff
@@ -35,23 +43,33 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
 
         builder.ConfigureAppConfiguration((_, configBuilder) =>
         {
+            // if the appSettings.json and appSettings.Development.json are messing too much with tests
+            // un-comment out the next line.
+            // configBuilder.Sources.Clear();
             configBuilder.AddConfiguration(_configuration);
         });
     }
 
     protected override IHost CreateHost(IHostBuilder builder)
     {
         builder.ConfigureContainer<ContainerBuilder>(ConfigureContainer);
+        builder.UseSerilog((context, services, configuration) =>
+        {
+            // this allows serilog log statements to appear in the test console runner
+            // e.g. database sql queries (issued by DbContext) can be observed this way in the test
+            configuration.WriteTo.Console(
+                restrictedToMinimumLevel: LogEventLevel.Debug,
+                formatProvider: CultureInfo.InvariantCulture,
+                outputTemplate: "[{Timestamp:HH:mm:ss} {Level:u3}] [{SourceContext}] {Message:lj}{NewLine}{Exception}");
```

Comment:
I think it's good to see the logs in the test output, but it will probably be too noisy without a override on the system namespaces. I would add some default overrides here, if possible.
These logs can be useful when investigating issues on the build server, for example, but if there is too much logging its more difficult to use the log file.